tests/analysis.spec.ts

```diff
@@ -252,7 +252,7 @@ describe('Functional test of analysis', () => {
 
         const sarifResults = extendedBundle.analysisResults.sarif;
 
-        expect(sarifResults.runs[0].tool.driver.rules).toHaveLength(5);
+        expect(sarifResults.runs[0].tool.driver.rules).toHaveLength(4);
```

Comment:
This test change seems unrelated to the sharding key changes, is it because our engine rules changed or perhaps mistakenly committed?